src/Plugin/Queue/QueueProcess.php

```diff
@@ -104,6 +104,7 @@ public function runPool(): void {
 	 *    buffer_table:string,
 	 *    destination_name:string,
 	 *    query:string,
+	 *    custom_mapping: string,
```

Comment:
We request string here but in parseMapping method returns false in case json failed to encode, is it normal?

Reply:
have never encountered issues with json_encode. In my opinion, it’s nearly impossible to encode an array with an error

Reply:
JSON easy to fail in case we accept user data. Does the encoded array get information from the outside of the world? User provide it? in case easy to trigger false so with unsupported Unicodes